homework_3.py

```diff
@@ -0,0 +1,17 @@
+number_1 = int(input("Say whole number from 1 to 10?"))
+number_2 = 2
+multiply_action = number_1 * number_2
+divide_action = number_1 / number_2
+plus_action = number_1 + number_2
+multiply_action = number_1 - number_2
+if number_1 in (2,4,6):
```

Comment:
I don't understand this decision.  Why our operation depends on number, You should ask the user what operation to use. 
operation = input("Enter your operation (+, -, /, *): ")
if operation == '+':
       result = number_1 + number_2
................